grpc/bcodes.go

```diff
@@ -21,13 +20,12 @@ import (
 // TODO(#2507): Deprecated, remove once boulder/errors code is deployed
 const (
 	MalformedRequestError = iota + 100
-	NotSupportedError
+	_
 	UnauthorizedError
 	NotFoundError
-	LengthRequiredError
+	_
 	RateLimitedError
-	BadNonceError
-	NoSuchRegistrationError
+	_
```

Comment:
Given you removed two errors here shouldn't there be an extra `_`?

Reply:
Also probably there should be a comment above this list that explains why the `_`s are here.

Reply:
Oh also also also: #2507 has been deployed, is it safe to just delete this code now?

Reply:
Good catch on the missing _. And yes, I think bcodes.go can be deleted entirely, but I didn't do it in this PR because I wanted to avoid making it too big.

Reply:
:+1: